src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -110,6 +110,8 @@ public void DynamicDirectory_Null()
         }
 
         [Fact]
+        // Not reliable when multiple executable assemblies are in the output directory.
```

Comment:
Instead of adding a comment here, should he add this as the attribute's string parameter?